tests/Auth0.ManagementApi.IntegrationTests/StatsTests.cs

```diff
@@ -18,10 +18,11 @@ public async Task Test_stats_sequence()
                 // Get stats for the past 10 days
                 var dailyStats = await apiClient.Stats.GetDailyStatsAsync(DateTime.Now.AddDays(-10), DateTime.Now);
                 dailyStats.Should().NotBeNull();
+                dailyStats.Count.Should().BeGreaterOrEqualTo(1);
 
                 // Get the active users
                 var activeUsers = await apiClient.Stats.GetActiveUsersAsync();
-                activeUsers.Should().BeGreaterOrEqualTo(0);
+                activeUsers.Should().BeGreaterOrEqualTo(1); // These integration tests themselves trigger non-zero values
```

Comment:
Were these not working before? If so, what changes in this PR changed this? 

Not saying you're doing anything wrong but, with these management API changes, I have definitely in the past lost sight of a functionality change because I thought that the integration tests were wonky ... turns out it was a potentially breaking change.

Reply:
This test was passing... but it wasn't a good test as 0 is the default value for an enum.  So if the payload returned bad data/didn't deserialize correctly this test would have passed when it shouldn't.

We can get away with `1` as by running the tests we effectively create an active user. In theory I guess this test could be run before the number gets updated depending on the order the tests run/consistency etc. I'll keep an eye on it.